app/common/Triggers.ts

```diff
@@ -10,6 +10,7 @@ export interface WebhookFields {
   url: string;
   eventTypes: Array<"add"|"update">;
   tableId: string;
+  columnIds: string;
```

Comment:
You may run `yarn build:prod` to update the `Triggers-ti.ts` file (whose classes are used as body validator for the APIs):

```
$ yarn build:prod
yarn run v1.22.21
$ buildtools/build.sh
+ tsc --build
+ buildtools/update_type_info.sh app
Updating app/common/Triggers-ti.ts from app/common/Triggers.ts
Updating app/plugin/CustomSectionAPI-ti.ts from app/plugin/CustomSectionAPI.ts
Updating app/plugin/GristData-ti.ts from app/plugin/GristData.ts
...
```